easybuild/easyconfigs/o/OVITO/OVITO-2.9.0-GCCcore-6.4.0-x86_64.eb

```diff
@@ -0,0 +1,29 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'PackedBinary'
+
+name = 'OVITO'
+version = '2.9.0'
+versionsuffix = '-x86_64'
+
+homepage = 'https://www.ovito.org/'
+
+description = """
+ OVITO is a scientific visualization and analysis software
+ for atomistic simulation data
+"""
+
+source_urls = [('https://www.ovito.org/download/%(version)s')]
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
+checksums = ['2eb484483dfe97003d0c1885899abdeb3dea00499a7a186b2f8811aa91d62052']
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
```

Comment:
@JackPerdue Why this `toolchain` if you're just using the binary release?

Reply:
Well when EB supports Robert's/Lmod's module version (greater/less than) ranges for the odd cases when you might want it for prebuild binaries.

I know this goes against what I like so much about EB... consistency.  But having that option would allow the user to load GCC >=5 of their choice and use it...

Reply:
OVITO-2.9.0-GCCcore-v5plus.eb ??? :)

Reply:
That's a whole can of worms you probably do not want to open, but I think we will support something like that someday so people can shoot theirselves in the foot.

The motivation for using `GCCcore` as a toolchain here is fine imho, maybe just include a comment above the `toolchain` line to explain it, since we will scratch our heads again later on this one, I'm sure...